examples/stocks/lib/stock_menu.dart

```diff
@@ -6,12 +6,14 @@ part of stocks;
 
 enum _MenuItems { autorefresh, autorefreshCheckbox, add, remove }
 
+const double _kMenuMargin = 16.0; // 24.0 on tablet
+
 Future showStockMenu(NavigatorState navigator, { bool autorefresh, ValueChanged onAutorefreshChanged }) async {
   switch (await showMenu(
     navigator: navigator,
     position: new MenuPosition(
-      right: sky.view.paddingRight,
-      top: sky.view.paddingTop
+      right: sky.view.paddingRight + _kMenuMargin,
+      top: sky.view.paddingTop + _kMenuMargin
```

Comment:
It seems like you want to say that the menu should appear the widget that triggered showing it and that it should additionally respect an additional margin around the application's body.